### ISSUE TITLE
This plugin didn't work on vscode.

### ISSUE THREAD

**Issue by zhiyingzzhou** [vuejs/eslint-plugin-vue#207]
**Tell us about your environment**

* **ESLint Version:4.8.0
* **eslint-plugin-vue Version:3.13.1
* **Node Version:6.11.3

**Please show your full configuration:**

```
module.exports = {
    "parser": "babel-eslint",
    "extends": [
        "airbnb-base",
        "plugin:vue/recommended"
    ],
    "env": {
      "browser": true,
      "node": true,
      "es6": true
      // "mocha": true
    },
    "plugins": [
      "import"
    ],
    "parserOptions": {
      "ecmaVersion": 6,
      "sourceType": "module",
      "ecmaFeatures": {
      }
      // "allowImportExportEverywhere": false,
      // "codeFrame": false
    }，
   “rules”: {
    }
}



```

**What did you do? Please include the actual source code causing the issue.**
I use eslint plugin on vscode.

```vue

export default {
  name: 'app',

};



```

**What did you expect to happen?**
It should report an error by vscode. 
**What actually happened? Please include the actual, raw output from ESLint.**
It didn't work on vscode.

**Comment by zhiyingzzhou** [comment]
I has resolve this issue.

**Comment by michalsnik** [comment]
Hi @zhiyingzzhou, I'm not sure what your problem is. If you solved it already, that's great. If not I think you have a problem with `babel-eslint`. We have a clear information in readme, that you shouldn't use parser directly, but use `parserOptions` instead (https://github.com/vuejs/eslint-plugin-vue/blob/94f387ed68149f8d12c1960d68cb5d9c376b9a89/README.md#what-is-the-use-the-latest-vue-eslint-parser-error) I hope this will solve your problem. If not - please feel free to reopen this issue :)
